README.md

```diff
@@ -11,9 +11,16 @@
 - [Stephane Reiss](https://github.com/T0RAT0RA)
 - [Jeremie Bardon](https://github.com/jeremiebardon)
 
-## Development
+## Satus
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/e2424f31-4626-483e-8e1c-e558c921c6d5/deploy-status)](https://app.netlify.com/sites/vuemontreal/deploys)
 
-
 # Check our projects if you want to know more about what next for vue montreal
+
+## Dev
+
+`npm ci && npm run dev`
+
+go to [localhost](http://localhost:3000)
+
+you can have access to our [stories](http://localhost:3000/.stories)
```

Comment:
Just to make things a bit clearer/cleaner: 

```suggestion
Go to: [http://localhost:3000](http://localhost:3000)

You can access our stories at: [http://localhost:3000/.stories](http://localhost:3000/.stories)
```

Reply:
yeap exactly.
About the file generate we have to figure out i didn't notice about the files generation. It's a little plugin that someone did maybe not the best one but i like the idea to not have two configs. One for project and one for storybook